includes/templates/admin/edit-story.php

```diff
@@ -31,6 +31,28 @@
 
 global $post_type, $post_type_object, $post;
 
+/**
+ * Filters the default content when creating a new story.
+ *
+ * @param string  $post_content Default post content.
+ * @param WP_Post $post         Post object.
+ */
+$post->post_content_filtered = (string) apply_filters( 'web_stories_default_content', $post->post_content_filtered, $post );
+
+add_filter(
+	'rest_prepare_' . \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG,
+	static function( WP_REST_Response $response ) use ( $post ) {
+		$data = $response->get_data();
+		if ( array_key_exists( 'story_data', $data ) ) {
+			$data['story_data'] = json_decode( $post->post_content_filtered, true );
+			$response->set_data( $data );
+		}
+
+		return $response;
+	},
+	10
+);
+
```

Comment:
This really doesn't feel like the best place for this code at all.

It should live here 
https://github.com/google/web-stories-wp/blob/7a202268654e2f4636d5ece8b53efc2e04bc66bd/includes/REST_API/Stories_Base_Controller.php#L100-L103

A new param should added to the REST API, like `popular_demo_data` or similar. 

This would also make the code more testable and trackable.